src/app/engine/audio-backend.service.ts

```diff
@@ -96,8 +96,13 @@ export class AudioBackendService {
 
   constructor(private http: HttpClient) {
     this.ready = false;
-    const hasWebM = MediaSource && MediaSource.isTypeSupported('audio/webm;codecs="vorbis"');
```

Comment:
How about changing this check to 

const hasWebM = (typeof MediaSource !== 'undefined') && MediaSource.isTypeSupported('audio/webm;codecs="vorbis"');

Perhaps this would do the trick as well? 

I'd prefer to avoid adding browser-specific detection code that may become obsolete once Apple adds this feature.

Reply:
Hi, thanks, sorry for the delay, I gonna check it, I'm fixing another bug at the same time, because with this fix in iOS we can hear the sound only with headphones... (https://stackoverflow.com/questions/21122418/ios-webaudio-only-works-on-headphones)